Userland/Libraries/LibWeb/Layout/Box.cpp

```diff
@@ -46,6 +46,16 @@ void Box::paint(PaintContext& context, PaintPhase phase)
         context.painter().draw_rect(enclosing_int_rect(margin_rect), Color::Yellow);
         context.painter().draw_rect(enclosing_int_rect(absolute_padding_box_rect()), Color::Cyan);
         context.painter().draw_rect(enclosing_int_rect(content_rect), Color::Magenta);
+
+        auto sizeTextRect = absolute_rect();
+        auto sizeText = String::formatted("{}x{}", content_rect.width(), content_rect.height());
```

Comment:
Please use `snake_case` for variable names.

Reply:
Hi, I corrected this and force pushed.